chess/capture-map.js

```diff
@@ -19,42 +19,44 @@ function renderGraph(data) {
   Plotly.newPlot(captureMap, graphData);
 }
 
-let matches = [];
-let squares = {};
-let coords = [];
-const regex = /x([a-hA-H][1-8])/gm;
+const regex = /x[a-hA-H][1-8]/gm;
```

Comment:
I removed the grouping here because we don't really need it - we can just skip the leading `x` via indexing.